Formula/redis.rb

```diff
@@ -12,6 +12,13 @@ class Redis < Formula
     sha256 "be205c8cecbf8691db5f06b60e9a5dccbfe1e6bef2e1676fe0dbdf109b2fdf9f" => :high_sierra
   end
 
+  patch do
+    # Fix for macOS 11
```

Comment:
```suggestion
    # Remove when upstream fix is released
    # Fix for macOS 11
```

A clear statement that this patch is temporary gives us a chance to search for, and follow up on, those cases.